RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2752,11 +2854,78 @@ boolean L11_palindome()
+		{
+			//TODO if no camera check if it is better to pull or go get one, than to find 4 more dudes and a Bob
+			if(item_amount($item[disposable instant camera]) == 0 || !auto_is_valid($item[disposable instant camera]))
+			{
+				dudesToDown = 10;	//if bob can't be photographed need to down more dudes
+			}
+		}
+
 		autoEquip($slot[acc3], $item[Talisman o\' Namsilat]);
 		if (handleFamiliar($familiar[Red-Nosed Snapper]))
 		{
 			auto_changeSnapperPhylum($phylum[dude]);
 		}
+		else if(auto_have_familiar($familiar[Nosy Nose]) && auto_is_valid($skill[Get a Good Whiff of This Guy]))
+		{
+			boolean noseDudesOn = true;
+			if(item_amount($item[Stunt Nuts]) == 0 && item_amount($item[Wet Stunt Nut Stew]) == 0)
+			{
+				//may want to use an item familiar first for stunt nuts
+				//unfortunately the sniff condition system means if taking the nose later after using different sniffs on a dude it will only be able to whiff on the same dude
+				int famWeightWithoutEq = familiar_weight(my_familiar()) + weight_adjustment() - numeric_modifier(equipped_item($slot[familiar]), "Familiar Weight");
+				int stuntNutDropModifierWithoutFamiliar = item_drop_modifier() + numeric_modifier("Food Drop") - numeric_modifier(my_familiar(), "Item Drop", famWeightWithoutEq, equipped_item($slot[familiar]));
+				if(stuntNutDropModifierWithoutFamiliar < 234)	//30% base drop chance
+				{
+					noseDudesOn = false;
+				}
+			}
+			if(noseDudesOn)
+			{
+				boolean whiffedBob = get_property("nosyNoseMonster").to_monster() == $monster[Racecar Bob] || get_property("nosyNoseMonster").to_monster() == $monster[Bob Racecar];
+				if(is_banished($monster[Flock of Stab-bats]) && is_banished($monster[Taco Cat]) && is_banished($monster[Tan Gnat]) && is_banished($monster[Evil Olive]))
+				{
+					//only dudes left already
+					noseDudesOn = false;
+				}
+				else if(get_property("palindomeDudesDefeated").to_int() >= dudesToDown)
+				{
+					if(dudesToDown >= 10 && whiffedBob)	//when looking for photograph of a dog without disposable instant camera
+					{	//the 10th or later dude must be a Bob, keep using the nose if it's tracking Bob
+						noseDudesOn = true;
+					}
+					else
+					{
+						//had enough dudes
+						noseDudesOn = false;
+					}
+				}
+				else if(get_property("palindomeDudesDefeated").to_int() == (dudesToDown - 1))
+				{
+					if(!whiffedBob)	//don't need to start sniffing the last dude
+					{
+						noseDudesOn = false;
+					}
+				}
+				else if(isSniffed($monster[Racecar Bob],$skill[Transcendent Olfaction]) || 
+				isSniffed($monster[Bob Racecar],$skill[Transcendent Olfaction]) || 
+				isSniffed($monster[Drab Bard],$skill[Transcendent Olfaction]) || 
+				getSniffer($monster[Racecar Bob], false) == $skill[Transcendent Olfaction] || 
+				getSniffer($monster[Bob Racecar], false) == $skill[Transcendent Olfaction])
+				{
+					//olfaction is or will be used and is probably powerful enough not to need weak nose tracking on
+					noseDudesOn = false;
```

Comment:
Would it be possible to both olfact and use nose in the same combat. Then next adv we would unequip nosey nose as one of the dudes has been olfacted. Maybe it is worth using the nose even if olfact has also been used?

Reply:
>Would it be possible to both olfact and use nose in the same combat. Then next adv we would unequip nosey nose as one of the dudes has been olfacted.

nose won't be coming along in combat under these conditions if Transcendent Olfaction will be used as it's first pick for getSniffer (insert nose picking joke here) so that wouldn't happen

adding nose to olfaction would make dude chances 3+3+1/3+3+3+1
so 7/10 instead of 6/9? barely a difference though I don't know why this is the only location I checked that